x-pack/legacy/plugins/snapshot_restore/public/app/services/validation/validate_policy.ts

```diff
@@ -28,57 +28,70 @@ export const validatePolicy = (policy: SlmPolicyPayload): PolicyValidation => {
       schedule: [],
       repository: [],
       indices: [],
+      minCount: [],
     },
   };
 
   if (isStringEmpty(name)) {
     validation.errors.name.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.nameRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.nameRequiredErroMessage', {
         defaultMessage: 'Policy name is required.',
       })
     );
   }
 
   if (isStringEmpty(snapshotName)) {
     validation.errors.snapshotName.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.snapshotNameRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.snapshotNameRequiredErrorMessage', {
         defaultMessage: 'Snapshot name is required.',
       })
     );
   }
 
   if (isStringEmpty(schedule)) {
     validation.errors.schedule.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.scheduleRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.scheduleRequiredErrorMessage', {
         defaultMessage: 'Schedule is required.',
       })
     );
   }
 
   if (isStringEmpty(repository)) {
     validation.errors.repository.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.repositoryRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.repositoryRequiredErrorMessage', {
         defaultMessage: 'Repository is required.',
       })
     );
   }
 
   if (config && typeof config.indices === 'string' && config.indices.trim().length === 0) {
     validation.errors.indices.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.indexPatternRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.indexPatternRequiredErrorMessage', {
         defaultMessage: 'At least one index pattern is required.',
       })
     );
   }
 
   if (config && Array.isArray(config.indices) && config.indices.length === 0) {
     validation.errors.indices.push(
-      i18n.translate('xpack.snapshotRestore.policyValidation.indicesRequiredError', {
+      i18n.translate('xpack.snapshotRestore.policyValidation.indicesRequiredErrorMessage', {
         defaultMessage: 'You must select at least one index.',
       })
     );
   }
 
+  if (
+    retention &&
+    retention.minCount &&
```

Comment:
Can `minCount` or `maxCount` ever be `0`?

Reply:
no, it must be at least `1`